user-interface/src/lib/components/MultiSelect.tsx

```diff
@@ -68,6 +75,21 @@ export default function MultiSelect(props: MultiSelectProps) {
     }),
   };
 
+  function clearValue() {
+    if (
+      multiSelectRef.current &&
+      Object.prototype.hasOwnProperty.call(multiSelectRef.current, 'clearValue')
```

Comment:
Since `hasOwnProperty` is on the object prototype we should be able to shorten this to `multiSelectRef.current.hasOwnProperty('clearValue')`. I believe we can also test if the clearValue function exists with a truthy expression like `&& multiSelectRef.current.clearValue`.